src/manifestv3.json

```diff
@@ -35,9 +38,19 @@
         "content/*",
         "options/*",
         "popup/*",
-        "background/*"
+        "background/*",
+        "specs/*"
       ],
       "matches": ["<all_urls>"]
     }
-  ]
+  ],
+  "content_security_policy": {
+    "extension_pages": "script-src 'self'; object-src 'self'"
+  },
```

Comment:
There is no need to generate a nonce?

Reply:
It's in fact, disallowed from what I tried. Leads to invalid CSP errors in Chrome (so does using a SHA).